### PR TITLE
CI Build - Prepare GCS build bucket for testing

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/22784

## Description
Splitting https://github.com/demisto/content/pull/7062 to 2 PRs - this is the first one

 - Copies master bucket to build bucket
 - Uploads relevant packs required for testing in the build

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Tests/Marketplace/upload_packs.py [hunk 1]
```diff
@@ -459,7 +458,7 @@ def option_handler():
                               "For more information go to: "
                               "https://googleapis.dev/python/google-api-core/latest/auth.html"),
                         required=False)
-    parser.add_argument('-i', '--id_set_path', help="The full path of id_set.json", required=False)
+    parser.add_argument('--id_set_path', help="The full path of id_set.json", required=False)
```
Comment (anara123) [inline Tests/Marketplace/upload_packs.py:461]: I think this change is not related to you PR

File: Tests/Marketplace/upload_packs.py [hunk 2]
```diff
@@ -552,12 +562,13 @@ def main():
             pack.cleanup()
             continue
 
-        task_status = pack.remove_unwanted_files()
+        task_status = pack.remove_unwanted_files(remove_test_playbooks)
```
Comment (anara123) [inline Tests/Marketplace/upload_packs.py:565]: this is also don't related to your PR

Comment (Itay4) [reply Tests/Marketplace/upload_packs.py:565]: actually i added this, it is a flag to set if not to remove test playbooks from the bucket
